\documentclass[12pt]{article}
\input{preamble.tex}

\begin{document}
\sloppy
\begin{center}\large{\textbf{Functional Activity of the Human Gut Microbiome in Colorectal Cancer}}\end{center}
\section*{Specific Aims}

Changes in the taxonomic composition and metabolic activity of the human gut microbiome have been observed in several disease including colorectal cancer (CRC).
Evidence of toxigenic activity by gut microbes implies that these changes are not only a response to disease, but perhaps drive disease progression.
Taxonomic composition studies have had mixed success in finding consistent patterns of microbiome changes between CRC and non-cancerous states;
the problem is challenging because interpersonal variability in taxonomic composition sometimes is greater than the variability between disease states.
It has been suggested that functional redundancy..

taxonomic composition of the microbiome as a diagnosistic tool for colorectal cancer, whether from 16S or whole metagenomes.
looking only at taxonomy may not be enough to understand the role and response of the gut microbiome in health and disease processes.
some studies couldn't find strong signal in 16S OTUs to differentiate CRC from normal, but found different metabolites [Weir 2013]

\subsection*{Aim 1. Assess the impact of functional redundancy of the gut microbiome on CRC classification.}
\textit{Hypothesis: Using functional gene profiles instead of only taxonomic profiles improves the classification modeling of samples as CRC or non-cancerous because of functional redundancy in the gut microbiome.}
\begin{compactenum}[A.]
    \item Build taxonomic profiles with OTUs from 16S rRNA gene sequences and build profiles of functional gene potential from metagenomes.
    \item Compare taxonomic composition to functional gene potential of microbiomes within and between disease states to determine presence and degree of functional redundancy.
    \item Build machine learning models to classify samples as CRC or non-cancerous with taxonomic composition, functional gene potential profiles, or both as model features and compare performance.
\end{compactenum}

\subsection*{Aim 2. Assess the impact of integrating active metabolites with functional gene potential on CRC classification.}
\textit{Hypothesis: Using active metabolic pathways confirmed with mass spectrometry instead of all potential metabolic pathways from metagenomes improves the classification modeling of samples as CRC or non-cancerous.}
\begin{compactenum}[A.]
    \item Annotate compounds from untargeted mass spectrometry with the GNPS database and select those known to be products of bacterial metabolic pathways with the MetaCyc database.
    \item Calculate the intersection of pathways associated with active metabolites and the pathways from functional potential profiles from metagenomes.
    \item Build machine learning models to classify samples as CRC or non-cancerous with all potential metabolic pathways or only confirmed active metabolic pathways as model features and compare performance.
\end{compactenum}


\pagebreak
\bibliographystyle{abbrv}
%\section*{\refname}
\footnotesize{
\bibliography{prelim.bib}
\par}
\end{document}