\documentclass[11pt]{article}
\input{submission/preamble.tex}

\begin{document} % 7 page limit
\sloppy
\begin{center}
\large{\textbf{
    Functional Activity of the Human Gut Microbiome to Classify Colorectal Cancer
}} \\
\vspace{11pt}
\small{
    Kelly L. Sovacool \\
    August 2020
}
\end{center}

\input{submission/aims.tex}

\section*{Background and Motivation} % 1/2 - 1 page

genetics and environmental factors explain only a small proportion of disease incidence, so we turn to the microbiome \cite{thomas_metagenomic_2019}.

``it is not possible to classify individuals as having healthy colons or screen relevant neoplasia using Bray- Curtis distances based on the 16S rRNA gene sequences collected from fecal samples (see Fig. S1 in the supplemental material). This variation is likely due to the ability of many bacterial populations to fill the same niche such that different populations cause the same disease in different individuals. Furthermore, a growing number of studies have shown that it is rare for a single bacterial species to be associated with a disease. Instead, subsets of the microbiome account for differences in health.'' \cite{topcuoglu_framework_2020} (Fig. \ref{fig:nmds_otu})

\begin{wrapfigure}{R}{0.5\textwidth}
    \begin{center}
        \includegraphics[width=0.5\textwidth]{figures/topcuoglu_s1.png}
        \caption{``Nonmetric multidimensional scaling (NMDS) ordination of Bray-Curtis distances. NMDS ordination relating the community structures of the fecal microbiota from 490 patients (261 patients with normal colonoscopies and 229 patients who have screen relevant neoplasias [SRNs]).'' \cite{topcuoglu_framework_2020}}
        \label{fig:nmds_otu}
    \end{center}
\end{wrapfigure}


previous study with n=10 CRC and normal. did 16S + untargeted metab. no sig difs in taxonomic comp. did find difs in metabolites. \cite{weir_stool_2013}

other studies found specifical potential functional pathways that are CRC biomarkers, especially choline metabolism pathway \cite{thomas_metagenomic_2019}.

While the concept of functional redundancy is well-developed in macroecology,
there does not seem to be a consensus on how to measure functional redundancy in culture-independent microbial communities,
where function is often assessed with meta'omic approaches \cite{ricotta_measuring_2016, louca_function_2018, heintz-buschart_human_2018, royalty_quantitative_2020}.

many published studies claiming to have found functional redundancies in microbial systems lack quantitative analyses of redundancy \cite{souza_metagenomic_2015, ferrer_microbiota_2013, galambos_genome-resolved_2019}.

no one agrees on exact definition of functional redundancy \cite{louca_function_2018, heintz-buschart_human_2018, tully_dynamic_2018, royalty_quantitative_2020}.
``Stability in ecosystem function with increasing microbial diversity is often considered an empirical indication of functional redundancy''
\cite{royalty_quantitative_2020}

we don't know enough about functional redundancy in the human gut microbiome \cite{heintz-buschart_human_2018}

``Community-level function is often more conserved than community composition, consistent with a functional repertoire ‘defining’ a niche and satisfied by different microbial assemblages.'' \cite{franzosa_species-level_2018}




trait contribution evenness (TCE): ``the evenness in relative contribution of that trait among taxa within the community... This definition has several appealing properties including: TCE is an extension of established diversity theory, functional redundancy measurements from communities with different richness and relative trait contribution by taxa are easily comparable, and any quantifiable trait data (genes copies, protein abundance, transcript copies, respiration rates, etc.) is suitable for analysis.'' \cite{royalty_quantitative_2020}

``Functional redundancy is a measure of the number of different populations within a community that are able to perform the same functions. Functional redundancy can increase functional resilience, in case perturbations affect the taxonomic community structure; this allows for a return to community function, and therefore can increase stability.'' \cite{heintz-buschart_human_2018}

``This functional redundancy is further reflected in the fraction of the observed microbial community capable of participating in each metabolic step, with no statistically significant difference between the boreholes, except for ammonia oxidation (Figure 6).'' (used Student's t-test and Wilcoxon rank sum) \cite{tully_dynamic_2018}


\section*{Significance} % 1/2 page


\section*{Research Design and Methods}

\subsection*{Aim 1. Functional redundancy of the gut microbiome}

\paragraph{1A) Build profiles of taxonomic composition and functional potential.}

16S rRNA gene sequencing was previously performed on stool samples from patients in the GLNE 007 cohort for classification modeling to detect CRC \cite{baxter_microbiota-based_2016}.
Since then, additional samples have been collected and sequenced, bringing the total dataset to 211 CRC and 223 non-cancerous samples.
Sequences will be processed with mothur according to the MiSeq SOP \cite{schloss_introducing_2009, kozich_development_2013}.
Briefly, processing steps include filtering for quality, removing chimeric sequences, clustering sequences into OTUs using the \textit{de novo} OptiClust method with a similarity threshold of 97\%,
and generating a table of OTU abundances by samples \cite{westcott_opticlust_2017}.
Abundances will be rarefied and converted to relative abundances to circumvent biases in sampling depth across samples.
This final OTU abundance table will serve as the taxonomic composition profiles of each community.

Whole metagenome shotgun sequencing will be performed and metagenomes will be processed with HUMAnN2 \cite{franzosa_species-level_2018} to characterize functional potential of the CRC and non-cancerous microbial communities.
Sequences will be filtered and trimmed for quality prior to processing with HUMAnN2.
HUMAnN2 uses MetaPhlan2 to screen sequences against a curated reference of 400,000 clade-specific marker genes to detect the microbial species present in each sample \cite{segata_metagenomic_2012}.
This strategy is assembly-free and saves considerable computational resources over assembly-based methods.
Next, sequences are mapped to annotated reference genomes to identify the gene families defined by Uniref90 and the metabolic pathways defined by MetaCyc \cite{caspi_metacyc_2018} that are encoded by each community.
The MetaCyc database contains pathways involving both primary and secondary metabolism and can be filtered by the domain of life.
MinPath pares down the list of metabolic pathways to the minimum set that can be explained by the genes encoded in each metagenome in order to avoid overestimating the pathways present \cite{ye_parsimony_2009}.
The end result is a conservative table of metabolic pathways encoded by each microbial community and their abundances.
As with OTU abundances, pathway abundances will be converted to relative abundances.
This table of pathway abundances will serve as the functional potential profiles of each community.


\paragraph{1B) Functional redundancy in CRC and non-cancerous gut microbiomes.}

Beta diversity is the difference in taxonomic composition between communities.
The Bray-Curtis dissimilarity index will be calculated on OTU relative abundances for all pairwise comparisons of samples as a measure of beta diversity \cite{bray_ordination_1957}.
For relative abundances, where the OTU abundances sum to 1 in each sample, the Bray-Curtis dissimilarity between a pair of samples is as follows:

$$b_{i i^\prime } = \frac{1}{2} \sum_{j}^J | r_{ij} - r_{i^\prime j} |$$

where the relative abundance of OTU $j$ in sample $i$ is $r_{ij}$ and the Bray-Curtis index between samples $i$ and $i^\prime$ is $b_{i i^\prime }$.
This can also be expressed as 1 minus the sum of the lesser abundances of OTUs that the samples have in common, which is equivalent to the above formula when applied to relative abundances \cite{greenacre_multivariate_2014}.
The range is 0 to 1, with 0 meaning the samples share all the same OTUs at equal abundance and 1 meaning the samples share no OTUs.
The Bray-Curtis index is preferred over the Jaccard index because the Jaccard index only takes presence and absence of OTUs into account, while Bray-Curtis also uses abundance data such that OTUs of higher abundance have greater influence over the diversity index than rarer OTUs.
A Non-metric Multidimensional Scaling (NMDS) plot will be created to visualize the Bray-Curtis dissimilarities.

Analysis of Similarity (ANOSIM) will be performed on the pairwise dissimilarity matrix of samples to test whether the similarity in taxonomic composition within disease states is greater than the similarity within disease states \cite{clarke_non-parametric_1993}.
To perform ANOSIM, all pairwise dissimilarities are ranked in order from most to least similar.
The average rank similarities of samples betweens sites ($\bar{r_B}$) and within sites ($\bar{r_W}$) are computed, and the test statistic $R$ is then calculated as:

$$R = \frac{ \bar{r_B} - \bar{r_W} }{ \frac{1}{4} n(n-1) }$$

with $n$ as the total number of samples.
The $R$ statistic ranges from -1 to 1, with 1 representing greatest similarity within sites, -1 representing greatest similarity between sites, and 0 representing no difference.
The null hypothesis is that the disease states are interchangeable, i.e. there is no significant difference in taxonomic composition between disease states.
A permutation test will then be performed to determine the significance.
The sample disease states will be permuted and $R$ will be calculated again for a random sample of 1,000 possible permutations.
Sampling permutations is necessary because there are $\frac{(kn)!}{n!^k k!}$ possible permutations of $k$ disease states for $n$ samples each;
for 2 disease states with 200 samples each that would be $5.1 \times 10^118$ permutations.
The permutated samples gives an estimate of the $R$ distribution under the null hypothesis,
thus fraction of permutation $R$s that are greater than or equal to the observed $R$ is the $P$ value.
Previous studies have not found significant differences in beta diversity between CRC and non-cancerous communities, and that result is expected here as well \cite{weir_stool_2013, hannigan_diagnostic_2018, topcuoglu_framework_2020}.

While diversity metrics are traditionally applied to taxa abundance data,
they can also be applied to other types of abundance data such as functional potential profiles \cite{franzosa_species-level_2018}.
Bray-Curtis dissimilarity will be calculated on metabolic pathway relative abundances for all pairwise comparisons of samples to measure the beta diversity of functional potential.
As with taxonomic beta diversity, significance of functional diversity will be assessed with ANOSIM and an NMDS plot will be created to visualize the dissimilarities.
If there is not a significant difference in taxonomic beta diversity between disease states,
while there is a significant difference in functional beta diversity between disease states,
functional redundancy \textit{within} disease states is a likely explanation.

\paragraph{1C) CRC classification models with taxonomic composition or functional potential.}

Binary random forest models will be built to classify samples as CRC or non-cancerous using OTU abundances, metabolic pathway abundances, or both as model features.
The random forest method has been found to perform well for microbiome-based classification problems because it can be used for non-linear data and accounts for interactions between features \cite{baxter_microbiota-based_2016}.
The dataset will be randomly split into 80\% training and 20\% testing sets, stratified to maintain the proportion of CRC to non-cancerous samples.
The mtry hyperparameter, which is the number of features used in each tree split, will be tuned to maximize the mean area under the receiver operating characteristic curve (AUC) with 5-fold cross-validation.
Each model will then be trained with the best mtry value and the test AUC will be calculated with the held-out test data.
These steps will be repeated for 100 iterations and the test AUCs will be recorded.
The statistical significance of differences in mean AUCs between the three types of models will be evaluated with a pairwise Wilcoxon test with Bonferroni-corrected P values for comparisons among the three models \cite{hannigan_diagnostic_2018}.
The Wilcoxon test is appropriate because for each model, there is a mean AUC for each data split, thus the mean AUCs are matched by data splits.
Permutation importance will be performed to determine which features (OTUs and pathways) have the greatest influence over model performance, which implies their importance as CRC biomarkers.
These methods have been lauded as best practices for building OTU-based machine learning models \cite{topcuoglu_framework_2020} and are currently being implemented in an R package (https://github.com/SchlossLab/mikRopML).
To reduce the runtime, these tasks will run in parallel where possible on the Great Lakes HPC cluster.

TODO RF model performance on 490 dataset with adenoma

\subsection*{Aim 2. Integrating active metabolites with functional gene potential}

\paragraph{2A) Annotate known products of bacterial metabolism from untargeted mass spectrometry.}

Untargeted liquid chromatography tandem mass spectrometry (LC-MS/MS) will be performed on stool samples to determine the functions actively performed by the bacterial community.
LC-MS/MS spectra will be processed with the Global Natural Products Social Molecular Networking (GNPS) a popular web-based tool for processing, annotating, and sharing tandem mass spectrometry data \cite{wang_sharing_2016}.
GNPS queries spectra against all reference spectra accumulated in GNPS libraries to find exact matches and annotate known compounds \cite{aron_reproducible_2020}.
As of 2016, GNPS had 18,163 compounds in its database, and trained users can contribute new spectra created from high quality standards.
The spectral search outputs structures of known annotated metabolites represented by spectra, which will be converted to International Chemical Identifiers (InChi) through the GNPS API for compatibility with the MetaCyc database.
It is important to note that stool samples contain metabolites which can be derived from host metabolism, microbial metabolism, both, or neither.
The functional potential from metagenomes provides an avenue to identify which metabolites are likely products of microbial metabolism.

\paragraph{2B) Find overlapping pathways from active metabolites and functional potential profiles.}

The IDs of metabolic products of all pathways encoded in the metagenomes of each microbial community (functional potential profiles) will be queried from the MetaCyc database to create a set of potential metabolites.
The set of potential metabolites will be intersected with the set of metabolites annotated in LC-MS/MS.
The intersection of these sets represents known metabolites which are 1) known to be products of bacterial metabolism in general and 2) capable of being produced by members of these specific microbial communities.
This set intersection would exclude metabolites that are not known to be capable of being produced by microbes, i.e. any metabolites that are only produced by human metabolism or from outside sources such as the host diet.
This method is inspired by AMON, which uses KEGG KOs rather than the MetaCyc database to putatively annotate the origins of metabolites in integrated metagenomic and metabolimic experiments \cite{shaffer_amon_2019}.
MetaCyc will be used here because it is already integrated with the HUMAnN2 tool for profiling functional potential, it contains more metabolic pathways than the KEGG database, and the KEGG database can no longer be downloaded in its entirety for free \cite{caspi_metacyc_2020}.

\paragraph{2C) CRC classification models with potential or confirmed active pathways.}

Binary random forest models to classify samples as CRC or non-cancerous will be built in a similar manner as described in Aim 1C, but with model features as potential metabolic pathways identified by HUMAnN2 or using only confirmed active metabolic pathways confirmed with LC-MS/MS.
Features will be coded as binary variables with 1 for pathway presence and 0 for pathway absence.
Presence/absence will be used rather than relative abundance because relative abundaces from potential pathways as determined by HUMAnN2 are fundamentally different from metabolite relative abundance as computed by the average area under the peak.
Best practices for model training and evaluation will be performed as described above including splitting training and testing data, tuning the mtry hyperparameter with 5-fold cross validation, calculating AUCs of each model on the held-out test data, and repeating these steps for 100 iterations.
The statistical significance of differences in mean AUC between the two types of models will be evaluated with a Wilcoxon test.
Finally, permutation importance will be performed to determine which metabolic pathways were most important for classification model performance.
If the mean AUC is significantly higher for models using only confirmed active pathways than those using functional pathways,
functional potential on its own is not sufficient to characterize true community function in CRC.
On the other hand, if models using potential pathways perform at least as well as models using only confirmed active pathways,
functional potential may compensate for metabolites missed by mass spectrometry.

\section*{Potential Outcomes and Conclusions}

limitation: genes with unknown functions

limitation: mass-spec features with unknown identity. Some have claimed that only ~2\% of spectra can be annotated in untargeted mass spectrometry experiments \cite{de_silva_illuminating_2015}.

limitation: not time-series

limitation: metabolites could be capable of being produced by microbes, but actually weren't being produced in the community at the time of sampling.

\pagebreak
\bibliographystyle{submission/nihunsrt} % downloaded from https://www.latextemplates.com/template/nih-grant-proposal
%\section*{\refname}
\footnotesize{
\bibliography{submission/prelim.bib}
\par}
\end{document}
