\documentclass[11pt]{article}
\input{submission/preamble.tex}

\begin{document} % 7 page limit
\sloppy
\begin{center}
\large{\textbf{
    Functional Activity of the Human Gut Microbiome to Classify Colorectal Cancer
}} \\
\vspace{11pt}
\small{
    Kelly L. Sovacool \\
    August 2020
}
\end{center}

\input{submission/aims.tex}

\section*{Background and Motivation} % 1/2 - 1 page


we don't know enough about functional redundancy in the human gut microbiome \cite{heintz-buschart_human_2018}

many published studies claiming to have found functional redundancies in microbial systems lack quantitative analyses of redundancy \cite{souza_metagenomic_2015, ferrer_microbiota_2013}.

no one agrees on exact definition of functional redundancy \cite{louca_function_2018, heintz-buschart_human_2018, tully_dynamic_2018, royalty_quantitative_2020}.
``Stability in ecosystem function with increasing microbial diversity is often considered an empirical indication of functional redundancy''
\cite{royalty_quantitative_2020}


trait contribution evenness (TCE): ``the evenness in relative contribution of that trait among taxa within the community... This definition has several appealing properties including: TCE is an extension of established diversity theory, functional redundancy measurements from communities with different richness and relative trait contribution by taxa are easily comparable, and any quantifiable trait data (genes copies, protein abundance, transcript copies, respiration rates, etc.) is suitable for analysis.'' \cite{royalty_quantitative_2020}

``Functional redundancy is a measure of the number of different populations within a community that are able to perform the same functions. Functional redundancy can increase functional resilience, in case perturbations affect the taxonomic community structure; this allows for a return to community function, and therefore can increase stability.'' \cite{heintz-buschart_human_2018}

``This functional redundancy is further reflected in the fraction of the observed microbial community capable of participating in each metabolic step, with no statistically significant difference between the boreholes, except for ammonia oxidation (Figure 6).'' (used Student's t-test and Wilcoxon rank sum) \cite{tully_dynamic_2018}

\section*{Significance} % 1/2 page


\section*{Research Design and Methods}

\subsection*{Aim 1. Functional redundancy of the gut microbiome}

\subsubsection*{1A) Build profiles of taxonomic composition and functional gene potential.}

16S, mothur

metagenomics, humann2

\subsubsection*{1B) Determine presence and degree of functional redundancy in CRC and non-cancerous gut microbiomes.}

How humann2 paper assessed fcnl potential w/in and b/e communities:
Calculate diversity metrics on function abundances to find "contributional diversity".
Alpha (within-sample): Gini-Simpson. Beta (between-sample): Bray-Curtis.
Fig 2A in \cite{franzosa_species-level_2018} plotted beta vs alpha for samples.
I'll do Wilcoxon rank-sum on Gini-Simpson, and ANOSIM + NMDS on Bray-Curtis distances with a post hoc multivariate Tukey test.'' \cite{hannigan_diagnostic_2018}

\subsubsection*{1C) Build and compare CRC classification models with taxonomic composition and functional gene potential.}

random forest or logistic regression

\subsection*{Aim 2. Integrating active metabolites with functional gene potential}

\subsubsection*{2A) Annotate compounds from untargeted mass spectrometry and select known products of bacterial metabolism.}

GNPS

inspired by AMON, but use MetaCyc instead of KEGG (MetaCyc has more pathways)

\subsubsection*{2B) Calculate the intersection of pathways from active metabolites and functional potential profiles.}

\subsubsection*{2C) Build and compare CRC classification models with all potential pathways or only confirmed active pathways.}

random forest or logistic regression

\section*{Potential Outcomes and Conclusions}

limitation: genes with unknown functions

limitation: mass-spec features with unknown identity

\pagebreak
\bibliographystyle{submission/nihunsrt} % downloaded from https://www.latextemplates.com/template/nih-grant-proposal
%\section*{\refname}
\footnotesize{
\bibliography{submission/prelim.bib}
\par}
\end{document}
