\documentclass[11pt]{article}
\input{submission/preamble.tex}
\begin{document} % 7 page limit
\sloppy

\begin{center}
\large{\textbf{
    Functional Activity of the Human Gut Microbiome to Classify Colorectal Cancer
}} \\
\vspace{11pt}
\small{
    Kelly L. Sovacool \\
    July 2020
}
\end{center}

\section*{Specific Aims} % 1 page

Changes in the taxonomic composition and metabolic activity of human microbiomes have been observed in several diseases.
In the case of colorectal cancer (CRC), evidence of toxigenic activity by gut microbes implies that these changes are not only a response to disease, but may also play a role in disease etiology.
Taxonomic composition is commonly defined by amplicon sequencing of the 16S rRNA gene and clustering sequences into Operational Taxonomic Units (OTUs).
Previous studies have built OTU-based machine learning models to classify stool samples as normal or cancerous, to serve as a less invasive diagnostic tool for CRC than colonoscopy.
Efforts to find consistent changes in taxonomic composition of microbiomes between normal and dysbiotic states have found mixed success, in part because interpersonal variability in taxonomic composition sometimes exceeds the variability between disease states.
Variability of microbiome composition between individuals with the same disease status may be explained by functional redundancy, where different microbial species carry out the same functions and thus can replace each other with little effect on the overall function of the community.

Sequencing whole metagenomes to identify the genes present and annotate known gene functions is commonly used to build a profile of functional potential of the microbiome.
Combining taxonomic composition from OTUs with functional potential from metagenomes allows one to characterize functional redundancy across communities, where communities with similar functional potential have different taxonomic composition.
Untargeted mass spectrometry can validate the functional potential characterized from metagenomics by identifying metabolites that are active in a community, thus painting a more precise picture of active microbial functions.
Here, I propose to investigate the impacts of taking functional redundancy and active metabolites into account on human stool sample classification for CRC diagnosis.


\subsection*{Aim 1. Assess the impact of functional redundancy of the gut microbiome on CRC classification.}
\textit{Hypothesis: Using functional gene profiles instead of only taxonomic profiles improves the classification modeling of samples as CRC or non-cancerous because of functional redundancy in the gut microbiome.}
\begin{compactenum}[A.]
    \item Build taxonomic profiles with OTUs from 16S rRNA gene sequences and build profiles of functional gene potential from metagenomes.
    \item Compare taxonomic composition to functional gene potential of microbiomes within and between disease states to determine presence and degree of functional redundancy.
    \item Build machine learning models to classify samples as CRC or non-cancerous with taxonomic composition, functional gene potential profiles, or both as model features and compare performance.
\end{compactenum}

\subsection*{Aim 2. Assess the impact of integrating active metabolites with functional gene potential on CRC classification.}
\textit{Hypothesis: Using active metabolic pathways confirmed with mass spectrometry instead of all potential metabolic pathways from metagenomes improves the classification modeling of samples as CRC or non-cancerous.}
\begin{compactenum}[A.]
    \item Annotate compounds from untargeted mass spectrometry with the GNPS database and select those known to be products of bacterial metabolic pathways with the MetaCyc database.
    \item Calculate the intersection of pathways associated with active metabolites and the pathways from functional potential profiles from metagenomes.
    \item Build machine learning models to classify samples as CRC or non-cancerous with all potential metabolic pathways or only confirmed active metabolic pathways as model features and compare performance.
\end{compactenum}

\section*{Background and Motivation} % 1/2 - 1 page

\section*{Significance} % 1/2 page

\section*{Research Design and Methods}

\subsection*{Aim 1. Functional redundancy of the gut microbiome}


\subsection*{Aim 2. Integrating active metabolites with functional gene potential}


\section*{Potential Outcomes and Conclusions}


\pagebreak
\bibliographystyle{abbrv}
\section*{\refname}
\footnotesize{
\bibliography{submission/prelim.bib}
\par}
\end{document}
